ui/src/scenes/Home/components/NeedsContainer/components/NeedsList/NeedsList.js

```diff
@@ -20,6 +23,31 @@ const NeedsListGroupItem = styled(ListGroupItem)`
   }
 `;
 
+const FlexDiv = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const RightMarginSpan = styled.span`
+  margin-right: 10px;
+`;
+
+const renderMissingRealizersAmount = (need) => {
+  let count = 0;
+  need.fulfilledBy.forEach((responsibility) => {
```

Comment:
Does this break when you add a new need and fulfilledBy is undefined?

Reply:
Also, you could use map rather than forEach to do the sum if you want more functional programming ;) That's very nitpicky though, so feel free to use forEach if you prefer :) There's probably a function in lodash that could do it elegantly too if you feel like it.

Reply:
Also, add fulfilledBy to propTypes :)

Reply:
Good catch. And you're right about the functional programming of course. 
All three things are now fixed.